types/models/clients.go

```diff
@@ -1,20 +1,47 @@
 package models
 
-import "time"
+import (
+	"time"
+
+	v1 "github.com/attestantio/go-eth2-client/api/v1"
+)
 
 // ClientsPageData is a struct to hold info for the clients page
 type ClientsPageData struct {
 	Clients     []*ClientsPageDataClient `json:"clients"`
 	ClientCount uint64                   `json:"client_count"`
+	PeerMap     *ClientPageDataPeerMap   `json:"peer_map"`
 }
 
 type ClientsPageDataClient struct {
-	Index       int       `json:"index"`
-	Name        string    `json:"name"`
-	Version     string    `json:"version"`
-	HeadSlot    uint64    `json:"head_slot"`
-	HeadRoot    []byte    `json:"head_root"`
-	Status      string    `json:"status"`
-	LastRefresh time.Time `json:"refresh"`
-	LastError   string    `json:"error"`
+	Index       int        `json:"index"`
+	Name        string     `json:"name"`
+	Version     string     `json:"version"`
+	HeadSlot    uint64     `json:"head_slot"`
+	HeadRoot    []byte     `json:"head_root"`
+	Status      string     `json:"status"`
+	LastRefresh time.Time  `json:"refresh"`
+	LastError   string     `json:"error"`
+	PeerId      string     `json:"peer_id"`
+	Peers       []*v1.Peer `json:"peers"`
```

Comment:
Please make a separate struct for `v1.Peer` with only the relevant information for the UI.
Small thing here, but UI models are cached in memory or in external cache, so they should be as small as possible :)

Reply:
Also has some security aspect of splitting up the view models to not accidentally exposing more than whats needed 👍